interface/billing/billing_report.php

```diff
@@ -65,1172 +62,1147 @@
- top.restoreSession();
- paturl = 'patient_file/summary/demographics_full.php?pid=' + pid;
- parent.left_nav.setPatient(pname,pid,pubpid,'',dobstr);
-    <?php if ($GLOBALS['new_tabs_layout']) { ?>
-  parent.left_nav.loadFrame('ens1', 'enc', 'patient_file/history/encounters.php?pid=' + pid);
-  parent.left_nav.loadFrame('dem1', 'pat', paturl);
-    <?php } else { ?>
-  var othername = (window.name == 'RTop') ? 'RBot' : 'RTop';
-  parent.frames[othername].location.href = '../' + paturl;
-    <?php } ?>
-}
-</script>
-<script language="javascript" type="text/javascript">
-EncounterDateArray=new Array;
-CalendarCategoryArray=new Array;
-EncounterIdArray=new Array;
-EncounterNoteArray=new Array;
-function SubmitTheScreen()
- {//Action on Update List link
-  if(!ProcessBeforeSubmitting())
-   return false;
-  top.restoreSession();
-  document.the_form.mode.value='change';
-  document.the_form.target='_self';
-  document.the_form.action='billing_report.php';
-  document.the_form.submit();
-  return true;
- }
-function SubmitTheScreenPrint()
- {//Action on View Printable Report link
-  if(!ProcessBeforeSubmitting())
-   return false;
-  top.restoreSession();
-  document.the_form.target='new';
-  document.the_form.action='print_billing_report.php';
-  document.the_form.submit();
-  return true;
- }
-  function SubmitTheEndDayPrint()
- {//Action on View End of Day Report link
-  if(!ProcessBeforeSubmitting())
-   return false;
-  top.restoreSession();
-  document.the_form.target='new';
-<?php if ($GLOBALS['use_custom_daysheet'] == 1) { ?>
-  document.the_form.action='print_daysheet_report_num1.php';
-<?php } ?>
-<?php if ($GLOBALS['use_custom_daysheet'] == 2) { ?>
-  document.the_form.action='print_daysheet_report_num2.php';
-<?php } ?>
-<?php if ($GLOBALS['use_custom_daysheet'] == 3) { ?>
-  document.the_form.action='print_daysheet_report_num3.php';
-<?php } ?>
-  document.the_form.submit();
-  return true;
- }
-function SubmitTheScreenExportOFX()
- {//Action on Export OFX link
-  if(!ProcessBeforeSubmitting())
-   return false;
-  top.restoreSession();
-  document.the_form.mode.value='export';
-  document.the_form.target='_self';
-  document.the_form.action='billing_report.php';
-  document.the_form.submit();
-  return true;
- }
-function TestExpandCollapse()
- {//Checks whether the Expand All, Collapse All labels need to be placed.If any result set is there these will be placed.
-    var set=-1;
-    for(i=1;i<=document.getElementById("divnos").value;i++)
-    {
-        var ele = document.getElementById("divid_"+i);
-        if(ele)
-        {
-        set=1;
-        break;
+    <?php //if (function_exists('html_header_show')) html_header_show(); ?>
+    <!--<link rel="stylesheet" href="<?php echo $css_header; ?>" type="text/css">
+    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker-2-5-4/build/jquery.datetimepicker.min.css">-->
+    <?php Header::setupHeader(['datetime-picker','font-awesome']);?>
```

Comment:
Change to (note font-awesome is included by default in this function):
``````
<?php Header::setupHeader('datetime-picker'); ?>